build/proxies-scenarios.yml

```diff
@@ -85,6 +94,10 @@ parameters:
     - displayName: h2 - h2
       arguments: --variable connections=28 --variable serverScheme=https --variable serverProtocol=http2 --variable downstreamScheme=https --variable downstreamProtocol=http2 --property serverProtocol=h2 --property downstreamProtocol=h2
       requiresServerHttp2: true
+      requiresDownstreamHttp2: true
+    - displayName: gRPC (h2 - h2)
+      arguments: --variable connections=1 --variable streams=100 --variable serverScheme=https --variable serverProtocol=grpc --variable downstreamScheme=https --variable downstreamProtocol=grpc
```

Comment:
I think you are missing `--property serverProtocol=grpc --property downstreamProtocol=grpc` or there will be no way to isolate those in the charts

Reply:
Fixed.